lib/begin.go

```diff
@@ -196,14 +197,31 @@ func (a *ACBuild) writeEmptyManifest() error {
 		return err
 	}
 
+	archvalue := runtime.GOARCH
+	if runtime.GOOS == "linux" && archvalue == "arm" {
+		elfFile, err := elf.Open("/proc/self/exe")
```

Comment:
Mostly because this was already written :p. I'll switch it over to use your method tomorrow morning.